src/frontend/src/components/Sidebar.js

```diff
@@ -112,10 +112,9 @@ const Sidebar = forwardRef((props, ref) => {
         step="0.01"
         id="threshold"
         defaultValue="0.1"
-        min="0.1"
-        max="0.9"
+        min="0.01"
```

Comment:
why don't we set min="0.0" and max="1.0"

Reply:
Good question! So the thing is that a lot of hexagons had a score<e-23 (that's an example number, just to say a very very small number, a model is rarely going to give you zero), so each prediction was storing waaaay too many hexagons. So I put a minimum threshold of 0.0001 (arbitrarily) to filter them out. Thus, min could not be zero because it wouldn't make sense. Since I set the step to be 0.01, the min had to be 0.01. The max I arbitrarily set to 0.99 (considering the 0.01 step) instead of 1 though. If we set it to 1, it will probably just show an empty map (the model is rarely going to give you an exact 0 or 1). I hope that clears it out, feel free to change it too if you think something different is more appropriate!

Reply:
the reasoning for min makes sense, I also noticed that species don't have much datapoints above 0.6-0.7 threshold, it isn't related to this ticket but something interesting